cl/recap/tasks.py

```diff
@@ -316,7 +316,8 @@ def process_recap_pdf(self, pk):
 
     # Do the file, finally.
     try:
-        file_contents = pq.filepath_local.read()
+        with pq.filepath_local.open("rb") as f:
+            new_sha1 = hashlib.file_digest(f, "sha1").hexdigest()
```

Comment:
Had to bump minimum python version requirement to 3.11 for `hashlib.file_digest` support.

Reply:
Fun. Want to remove the old crypto functions we had for this purpose now that Python caught up with us? :)

Reply:
Hmm, those are getting used in a bunch of places where file_digest may not always be a suitable replacement, probably best left as a followup.

Reply:
Oh, actually, I was looking at sha1_of_file, but whatever, we can get back to that, yes.